src/components/base/BaseCard.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-sheet>
+    <v-card v-bind="$attrs" :class="{ 'c-card': !embedded }" :flat="embedded">
+      <slot />
+    </v-card>
+
+    <v-divider v-if="divider" class="mt-3" />
+  </v-sheet>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component({
+  name: 'BaseCard',
+})
+export default class BaseCard extends Vue {
+  @Prop({ default: false }) private readonly embedded!: false;
```

Comment:
`: boolean`?

Reply:
yeah should boolean, I copy from other component